content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using AWS Remotes in DVC
+date: 2022-05-17
+description: >
+  We're going to set up an AWS S3 remote in a DVC project.
+descriptionLong: >
+  It can be tricky setting up a remote to handle data versioning in DVC so we're
+  going to go through a tutorial for doing this with AWS.
+picture: 2022-05-17/aws-in-dvc.png
+pictureComment: Using AWS Remotes in DVC
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/preventing-stale-models-in-production/1137
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - DVC Remotes
+  - Collaboration
+---
+
+When you’re working on a data science project that has huge datasets, it’s
+common to store them in cloud storage. You’ll also be working with different
+versions of the same datasets to train a model, so it’s crucial to have a tool
+that enables you to do this quickly and easily. That’s why we’re going to do a
+quick walkthrough of how to set up a remote in an AWS S3 bucket and handle data
+versioning with [DVC](https://dvc.org/doc).
+
+We’ll start by creating a new S3 bucket in our AWS account, then we’ll show how
+you can add DVC to your project, and finally, we’ll make updates to the dataset
+with DVC commands. We’ll be working with
+[this repo](https://github.com/iterative/stale-model-example) if you want an
+example to play with. By the time you finish, you should be able to create this
+setup for any ML project using an AWS remote.
```

Comment:
This could be a note or [admonition](https://github.com/iterative/dvc.org/issues/3382) (visually separated from the main text of the intro).

Also, ML -> machine learning (for SEO)